third_party/packages/perfetto_compiled/README.md

```diff
@@ -27,4 +27,16 @@ of ad hoc updates, this output should be refreshed at least once per quarter.
 
 To update the Perfetto build output, run the `update_perfetto.sh` script located
 in `devtools/tools/`. Be sure that all DevTools-authored files under
-`assets/perfetto` are not deleted by mistake. 
\ No newline at end of file
+`assets/perfetto` are not deleted by mistake.
+
+## How to update this build for local changes in the Perfetto codebase?
+
```

Comment:
Is it Perfetto repo? If yes, make it a link.

Reply:
Added links to perfetto build instructions